src/splitInteger.test.js

```diff
@@ -4,18 +4,23 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-
+  expect(splitInteger(6, 2)).toEqual([3, 3]);
+  expect(splitInteger(12, 4)).toEqual([3, 3, 3, 3]);
```

Comment:
The expected result for `splitInteger(12, 4)` is incorrect. According to the task description, the array should be sorted in ascending order and the difference between the max and min number should be <= 1. Please verify the implementation of `splitInteger` to ensure it meets these requirements.